### PR TITLE
Rewrite of Toon integration

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

Documentation that matches the rewrite of the Toon integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/36952
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/toon.markdown [hunk 1]
```diff
@@ -14,15 +15,29 @@ ha_codeowners:
...
-For the `toon` integration to work, you'll need an active Toon subscription with Eneco and a Toon API developer account.
+This integration adds a climate device for your Toon thermostat, some switches
+allowing you to control the program and holiday mode of the thermostats as well.
+
+Sensors for energy, power and gas consumption, sensors for solar production and
+several binary sensors for things like boiler burner on/off, hot tap water and
+boiler health status.
+
+For the Toon integration to work, you'll need an active Toon subscription with
+and a Toon API developer account.
```
Comment (scheric) [inline source/_integrations/toon.markdown]: A minor suggestion:
```suggestion
For the Toon integration to work, you'll need an active Toon subscription 
and a Toon API developer account.
```

**Review by scheric — COMMENTED** [review]
## Thanks for the great work on this component. 👍

#### some things that happened during configuration
I did have a little problem during the configuration in Home Assistant on my virtual machine. 
I had a clean install that did not have the internal URL configured. (I do not know it that is standard.) 
I got this result:

 login error
------------
On my active Home Assistant that has a base URL configured in YAML threw a `500 Internal Server Error` but it did configure properly.

File: source/_integrations/toon.markdown (hunk 1 above)
Comment (frenck) [reply source/_integrations/toon.markdown]: Thanks, @scheric 👍

**Comment by frenck** [comment]
@scheric OAuth in Home Assistant uses the internal URL by default. If that is not set, it will be the IP Address of the machine + port (e.g., `http://192.168.4.5:8123`). You need to use that one. However, this is a general HA issue which (imho) needs addressing 😉

**Review by balloob — APPROVED** [review]

**Pull request merged**